obspy/io/nordic/tests/test_nordic.py

```diff
@@ -456,8 +456,24 @@ def test_read_picks_across_day_end(self):
             self.assertTrue(pick.time in pick_times)
         self.assertEqual(event_2.origins[0].time, event.origins[0].time)
 
-
-def test_similarity(event_1, event_2):
+    def test_distance_conversion(self):
+        """Check that distances are converted properly."""
+        testing_path = os.path.join(self.testing_path, '01-0411-15L.S201309')
+        cat = read_events(testing_path)
+        event = cat[0]
+        self.assertEqual(sorted(event.origins[0].arrivals,
+                                key=lambda x: x.distance)[0].distance,
+                         0.035972864236749225)
```

Comment:
This should probably be an "assertAlmostEqual" just to deal with floating point inaccuracies. Tests seem to pass fine but it cannot hurt to be safe.